velox/exec/StreamingWindow.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/exec/Operator.h"
+#include "velox/exec/RowContainer.h"
+#include "velox/exec/Window.h"
+#include "velox/exec/WindowFunction.h"
+#include "velox/exec/WindowPartition.h"
+
+namespace facebook::velox::exec {
+
+/// This operator differs from the Window operator in that it does not require
+/// sorting the input vector or caching all the input data. Instead, it groups
+/// the input batch, performs calculations on first n-1 partitions, and carries
+/// forward the last partition to the next input batch.
+class StreamingWindow : public Window {
```

Comment:
Do we need to update [operators.rst](https://github.com/facebookincubator/velox/blob/main/velox/docs/develop/operators.rst) for this operator?

Reply:
Yes. Thanks your reminder.